modules/cross_account_principal_association/versions.tf

```diff
@@ -3,8 +3,9 @@ terraform {
 
```

Comment:
This actually also requires updating the terraform required_version to `>= 0.15`